test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3499,5 +3499,63 @@ public void testScoreboardHTMLLocations() throws Exception {
         assertEquals("Public HTML Directory", "public_html_dir", publicDir);
 
     }
+    
+    
+    public void testTenProbsisStopOnFirstFailedTestCase() throws Exception {
+        
+        String sampleName = "tenprobs";
+        IInternalContest contest  = fullLoadSampleContest(sampleName);
+        assertNotNull(contest);
+
+        Problem[] problems = contest.getProblems();
+        assertEquals("Num problems ",10, problems.length);
+        
+        for (Problem problem : problems) {
+            assertTrue(problem.getShortName()+" stop on first ", problem.isStopOnFirstFailedTestCase());
+        }
+        
+    }
+    
+    public void tesproblemflagtestStopOnFirstFailedTestCase() throws Exception {
+        
+        String sampleName = "problemflagtest";
+        IInternalContest contest  = fullLoadSampleContest(sampleName);
+        assertNotNull(contest);
+
+        Problem[] problems = contest.getProblems();
+        assertEquals("Num problems ",10, problems.length);
+        
+        for (Problem problem : problems) {
+            assertTrue(problem.getShortName()+" stop on first ", problem.isStopOnFirstFailedTestCase());
+        }
+        
+    }
+    
+    
+    
+    public void testvaltesttStopOnFirstFailedTestCase() throws Exception {
```

Comment:
This seems like a weird name for a method... why is there a "double-t" in the name?

Reply:
renamed method and pushed.

Reply:
I'm still confused about this method name:  why does it refer to "valtest"?  And again, why does it have a "double-t" in the name  (...valtesttStop...)?

Reply:
... I'm letting this go....